### ISSUE TITLE
Delayed launch on Windows 11 (compared to Windows 10)

### ISSUE THREAD

**Issue by oscarhermoso** [bitwarden/clients#11679]
### Steps To Reproduce

1. On Windows 11, without the Bitwarden desktop client running,
2. Type `Super`, `b`, `i`, `t`, `w`, `Enter` to open the Start Menu and launch Bitwarden

### Expected Result

Bitwarden Desktop app launches quickly, fully loading within 2-3 seconds. (same as Windows 10)

### Actual Result

However, it takes about 5 seconds for the Desktop app to load, and another 10 seconds before the password field is visible.

### Screenshots or Videos

Apologies for the shaky video, the Bitwarden desktop client isn't captured on OBS

https://github.com/user-attachments/assets/3a689528-c73e-47ab-94a0-b1f0b076a316

### Additional Context

Work arounds for anyone else running into this - install the PWA for [Bitwarden Vault](https://vault.bitwarden.com/#/login) instead of using the Desktop client

### Operating System

Windows

### Operating System Version

Windows 11 Enterprise 23H2, OS Build 22631.3447

### Installation method

Other

### Build Version

Version 2024.10.1 Shell 32.1.1 Renderer 128.0.6613.137 Node 20.17.0 Architecture x64

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by sammbw** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by oscarhermoso** [comment]
Hi @sammbw,

Here is a fresh dump of `app.log`, showing the detail of only one app launch. Let me know if there is any other information that I can capture to assist - eg. should I launch the app with some flags to add additional detail?

```
[2024-10-24 08:26:38.490] [info]  State version: 68
[2024-10-24 08:26:38.494] [info]  Migrator MinVersionMigrator (to version 3) should migrate: false - up
[2024-10-24 08:26:38.495] [info]  Migrator RemoveEverBeenUnlockedMigrator (to version 4) should migrate: false - up
[2024-10-24 08:26:38.495] [info]  Migrator AddKeyTypeToOrgKeysMigrator (to version 5) should migrate: false - up
[2024-10-24 08:26:38.495] [info]  Migrator RemoveLegacyEtmKeyMigrator (to version 6) should migrate: false - up
[2024-10-24 08:26:38.496] [info]  Migrator MoveBiometricAutoPromptToAccount (to version 7) should migrate: false - up
[2024-10-24 08:26:38.497] [info]  Migrator MoveStateVersionMigrator (to version 8) should migrate: false - up
[2024-10-24 08:26:38.504] [info]  Migrator MoveBrowserSettingsToGlobal (to version 9) should migrate: false - up
[2024-10-24 08:26:38.505] [info]  Migrator EverHadUserKeyMigrator (to version 10) should migrate: false - up
[2024-10-24 08:26:38.506] [info]  Migrator OrganizationKeyMigrator (to version 11) should migrate: false - up
[2024-10-24 08:26:38.507] [info]  Migrator MoveEnvironmentStateToProviders (to version 12) should migrate: false - up
[2024-10-24 08:26:38.508] [info]  Migrator ProviderKeyMigrator (to version 13) should migrate: false - up
[2024-10-24 08:26:38.509] [info]  Migrator MoveBiometricClientKeyHalfToStateProviders (to version 14) should migrate: false - up
[2024-10-24 08:26:38.510] [info]  Migrator FolderMigrator (to version 15) should migrate: false - up
[2024-10-24 08:26:38.510] [info]  Migrator LastSyncMigrator (to version 16) should migrate: false - up
[2024-10-24 08:26:38.511] [info]  Migrator EnablePasskeysMigrator (to version 17) should migrate: false - up
[2024-10-24 08:26:38.511] [info]  Migrator AutofillSettingsKeyMigrator (to version 18) should migrate: false - up
[2024-10-24 08:26:38.512] [info]  Migrator RequirePasswordOnStartMigrator (to version 19) should migrate: false - up
[2024-10-24 08:26:38.512] [info]  Migrator PrivateKeyMigrator (to version 20) should migrate: false - up
[2024-10-24 08:26:38.513] [info]  Migrator CollectionMigrator (to version 21) should migrate: false - up
[2024-10-24 08:26:38.513] [info]  Migrator CollapsedGroupingsMigrator (to version 22) should migrate: false - up
[2024-10-24 08:26:38.514] [info]  Migrator MoveBiometricPromptsToStateProviders (to version 23) should migrate: false - up
[2024-10-24 08:26:38.514] [info]  Migrator SmOnboardingTasksMigrator (to version 24) should migrate: false - up
[2024-10-24 08:26:38.515] [info]  Migrator ClearClipboardDelayMigrator (to version 25) should migrate: false - up
[2024-10-24 08:26:38.515] [info]  Migrator RevertLastSyncMigrator (to version 26) should migrate: false - up
[2024-10-24 08:26:38.515] [info]  Migrator BadgeSettingsMigrator (to version 27) should migrate: false - up
[2024-10-24 08:26:38.516] [info]  Migrator MoveBiometricUnlockToStateProviders (to version 28) should migrate: false - up
[2024-10-24 08:26:38.516] [info]  Migrator UserNotificationSettingsKeyMigrator (to version 29) should migrate: false - up
[2024-10-24 08:26:38.516] [info]  Migrator PolicyMigrator (to version 30) should migrate: false - up
[2024-10-24 08:26:38.517] [info]  Migrator EnableContextMenuMigrator (to version 31) should migrate: false - up
[2024-10-24 08:26:38.517] [info]  Migrator PreferredLanguageMigrator (to version 32) should migrate: false - up
[2024-10-24 08:26:38.517] [info]  Migrator AppIdMigrator (to version 33) should migrate: false - up
[2024-10-24 08:26:38.517] [info]  Migrator DomainSettingsMigrator (to version 34) should migrate: false - up
[2024-10-24 08:26:38.518] [info]  Migrator MoveThemeToStateProviderMigrator (to version 35) should migrate: false - up
[2024-10-24 08:26:38.518] [info]  Migrator VaultSettingsKeyMigrator (to version 36) should migrate: false - up
[2024-10-24 08:26:38.518] [info]  Migrator AvatarColorMigrator (to version 37) should migrate: false - up
[2024-10-24 08:26:38.519] [info]  Migrator TokenServiceStateProviderMigrator (to version 38) should migrate: false - up
[2024-10-24 08:26:38.519] [info]  Migrator MoveBillingAccountProfileMigrator (to version 39) should migrate: false - up
[2024-10-24 08:26:38.519] [info]  Migrator OrganizationMigrator (to version 40) should migrate: false - up
[2024-10-24 08:26:38.519] [info]  Migrator EventCollectionMigrator (to version 41) should migrate: false - up
[2024-10-24 08:26:38.520] [info]  Migrator EnableFaviconMigrator (to version 42) should migrate: false - up
[2024-10-24 08:26:38.520] [info]  Migrator AutoConfirmFingerPrintsMigrator (to version 43) should migrate: false - up
[2024-10-24 08:26:38.520] [info]  Migrator UserDecryptionOptionsMigrator (to version 44) should migrate: false - up
[2024-10-24 08:26:38.520] [info]  Migrator MergeEnvironmentState (to version 45) should migrate: false - up
[2024-10-24 08:26:38.521] [info]  Migrator DeleteBiometricPromptCancelledData (to version 46) should migrate: false - up
[2024-10-24 08:26:38.521] [info]  Migrator MoveDesktopSettingsMigrator (to version 47) should migrate: false - up
[2024-10-24 08:26:38.521] [info]  Migrator MoveDdgToStateProviderMigrator (to version 48) should migrate: false - up
[2024-10-24 08:26:38.521] [info]  Migrator AccountServerConfigMigrator (to version 49) should migrate: false - up
[2024-10-24 08:26:38.522] [info]  Migrator KeyConnectorMigrator (to version 50) should migrate: false - up
[2024-10-24 08:26:38.522] [info]  Migrator RememberedEmailMigrator (to version 51) should migrate: false - up
[2024-10-24 08:26:38.522] [info]  Migrator DeleteInstalledVersion (to version 52) should migrate: false - up
[2024-10-24 08:26:38.523] [info]  Migrator DeviceTrustServiceStateProviderMigrator (to version 53) should migrate: false - up
[2024-10-24 08:26:38.523] [info]  Migrator SendMigrator (to version 54) should migrate: false - up
[2024-10-24 08:26:38.523] [info]  Migrator MoveMasterKeyStateToProviderMigrator (to version 55) should migrate: false - up
[2024-10-24 08:26:38.523] [info]  Migrator AuthRequestMigrator (to version 56) should migrate: false - up
[2024-10-24 08:26:38.524] [info]  Migrator CipherServiceMigrator (to version 57) should migrate: false - up
[2024-10-24 08:26:38.524] [info]  Migrator RemoveRefreshTokenMigratedFlagMigrator (to version 58) should migrate: false - up
[2024-10-24 08:26:38.524] [info]  Migrator KdfConfigMigrator (to version 59) should migrate: false - up
[2024-10-24 08:26:38.524] [info]  Migrator KnownAccountsMigrator (to version 60) should migrate: false - up
[2024-10-24 08:26:38.525] [info]  Migrator PinStateMigrator (to version 61) should migrate: false - up
[2024-10-24 08:26:38.525] [info]  Migrator VaultTimeoutSettingsServiceStateProviderMigrator (to version 62) should migrate: false - up
[2024-10-24 08:26:38.526] [info]  Migrator PasswordOptionsMigrator (to version 63) should migrate: false - up
[2024-10-24 08:26:38.526] [info]  Migrator GeneratorHistoryMigrator (to version 64) should migrate: false - up
[2024-10-24 08:26:38.526] [info]  Migrator ForwarderOptionsMigrator (to version 65) should migrate: false - up
[2024-10-24 08:26:38.527] [info]  Migrator MoveFinalDesktopSettingsMigrator (to version 66) should migrate: false - up
[2024-10-24 08:26:38.527] [info]  Migrator RemoveUnassignedItemsBannerDismissed (to version 67) should migrate: false - up
[2024-10-24 08:26:38.527] [info]  Migrator MoveLastSyncDate (to version 68) should migrate: false - up
[2024-10-24 08:26:43.689] [info]  Checking for update
[2024-10-24 08:26:47.345] [info]  Update for version 2024.10.1 is not available (latest version: 2024.10.1, downgrade is disallowed).
[2024-10-24 08:26:49.923] [info]  State version: 68

````

**Comment by oscarhermoso** [comment]
Issue is no longer occurring - seems it may have been a Windows bug?

Bitwarden desktop version has not changed, but my operating system has been updated to Windows 11 Enterprise 23H2, OS Build 22631.3447, Windows Feature Experience Pack 1000.22700.1041.0.

Thankyou for any time spent investigating.
